packages/service-library/src/servicelib/status_utils.py

```diff
@@ -0,0 +1,94 @@
+""" Usage
+
+    - on aiohttp services
+        from servicelib.aiohttp import status
+        from servicelib.status_utils import is_success
+
+        assert is_success(status.HTTP_200_OK)
+
+
+    - on fastapi services
+
+        from fastapi import status
+        from servicelib.status_utils import is_success
+
+        assert is_success(status.HTTP_200_OK)
+
+    NOTE: https://github.com/encode/httpx/blob/master/httpx/_status_codes.py
+"""
+
+import types
+from collections.abc import Callable
+from http import HTTPStatus
+
+_INVALID_STATUS_CODE_MSG = "INVALID_STATUS_CODE"
+
+
+def get_display_name(status_code: int) -> str:
+    """
+    Returns display name given a status code, e.g.
+
+        get_display_name(200) == "HTTP_200_OK"
+        get_display_name(status.HTTP_200_OK) == "HTTP_200_OK"
+    """
+    try:
+        code = HTTPStatus(status_code)
+        return f"HTTP_{status_code}_{code.name}"
+    except ValueError:
+        if status_code == 306:  # noqa: PLR2004
+            return "HTTP_306_RESERVED"
+        return _INVALID_STATUS_CODE_MSG
+
+
+def is_informational(status_code: int) -> bool:
+    """
+    Returns `True` for 1xx status codes, `False` otherwise.
+    """
+    return 100 <= status_code <= 199  # noqa: PLR2004
```

Comment:
@bisgaard-itis These are some arguments to support this approach:
1. because this module should be accessible to both fastapi and aiohttp-based services (therefore it should never import from the `fastapi` or `aiohptt` subpackage). 
2. because this is a code range and there is even no status code to represent e.g. 199. 
3. because it is more readable a number range than something like `status.HTTP_100_XX < status_code < 199`